apps/dotcom/client/src/tla/components/TlaFileShareMenu/TlaFileShareMenu.tsx

```diff
@@ -70,9 +69,17 @@ export function TlaFileShareMenu({
 					{app ? (
 						<TlaTabsRoot activeTab={shareMenuActiveTab} onTabChange={handleTabChange}>
 							<TlaTabsTabs>
-								<TlaTabsTab id="share">{raw('Invite')}</TlaTabsTab>
-								<TlaTabsTab id="export">{raw('Export')}</TlaTabsTab>
-								{showPublishTab && <TlaTabsTab id="publish">{raw('Publish')}</TlaTabsTab>}
+								<TlaTabsTab id="share">
+									<F defaultMessage="Invite" />
```

Comment:
Just as a curiosity, why `F`?

Reply:
it's short for `FormattedMessage` which, I dunno, just seems really long/a mouthful.